projects/packages/publicize/src/class-publicize.php

```diff
@@ -653,8 +653,8 @@ public function test_connection( $service_name, $connection ) {
 
 		$error_data = array(
 			'user_can_refresh' => $user_can_refresh,
-			'refresh_text'     => $refresh_text,
-			'refresh_url'      => $refresh_url,
+			'refresh_text'     => $refresh_text ?? '',
+			'refresh_url'      => $refresh_url ?? '',
```

Comment:
Why not this, that better matches the current behavior?

```suggestion
			'refresh_text'     => $refresh_text ?? null,
			'refresh_url'      => $refresh_url ?? null,
```